nixos/modules/services/monitoring/netdata.nix

```diff
@@ -247,7 +247,9 @@ in {
         ++ lib.optional cfg.python.enable (pkgs.python3.withPackages cfg.python.extraPackages)
         ++ lib.optional config.virtualisation.libvirtd.enable config.virtualisation.libvirtd.package
         ++ lib.optional config.virtualisation.docker.enable config.virtualisation.docker.package
-        ++ lib.optionals config.virtualisation.podman.enable [ pkgs.jq config.virtualisation.podman.package ];
+        ++ lib.optionals config.virtualisation.podman.enable [ pkgs.jq config.virtualisation.podman.package ]
+        # Collector's `binary_path` config option has to be set to `zpool` for this to work.
```

Comment:
Default `binary_path` is [`/usr/bin/zpool`](https://github.com/netdata/netdata/blob/ce8b97706133fa2b4debffaf3d04d66d9b9024b3/src/go/collectors/go.d.plugin/modules/zfspool/zfspool.go#L31)

Reply:
I think the comment can be omitted.